src/iceberg/table_identifier.h

```diff
@@ -35,6 +37,15 @@ struct ICEBERG_EXPORT Namespace {
   std::vector<std::string> levels;
 
   bool operator==(const Namespace& other) const { return levels == other.levels; }
+
+  std::string ToString() const {
+    std::ostringstream oss;
+    for (size_t i = 0; i < levels.size(); ++i) {
```

Comment:
`formatter_internal.h` has a `FormatRange` to implement this. Using it requires adding a table_identifiler.cc since we cannot export internal header here.